src/util/VRControllerManager.js

```diff
@@ -271,4 +272,16 @@ x3dom.VRControllerManager.prototype._updateControllerModels = function ( vieware
 
         this.rightTransform.setAttribute( "matrix", matrix.toString() );
     }
+};
+
+x3dom.VRControllerManager.prototype._getViewAreaZoom = function ( viewarea )
+{
+    var navi = viewarea._scene.getNavigationInfo();
+    var viewpoint = viewarea._scene.getViewpoint();
+    var d = ( viewarea._scene._lastMax.subtract( viewarea._scene._lastMin ) ).length();
+    d = Math.min( d, viewpoint.getFar() );
+    d = ( ( d < x3dom.fields.Eps ) ? 1 : d ) * navi._vf.speed;
+    const zoomAmount = 1;
```

Comment:
@andreasplesch is this logic fine?
```
const fps = this._doc._x3dElem.runtime.getFPS();
x3dom.debug.logInfo( "fps: " + fps );
const zoomAmount = 1 / ( fps / 60 );

return d * ( zoomAmount ) / viewarea._height;
```
It seems to work in a similar way.

Reply:
Yes, I think RAF in VR is targeting 60 fps, so that looks good. Just simplify to
```js
const zoomAmount = 60 / fps ;
```
Does that scaling approximate a zoom speed of 1 m/s at nav speed of 1 when moving ?

Reply:
I am not sure if its approx to 1 m/s as the zoom also takes viewpoint far into consideration also.

Reply:
Good point. Let's just target zooming to be comfortable with default settings in a say room sized scene.

Reply:
I think its comfortable at room level scene.

Reply:
@andreasplesch however I think the Zoom may feel comfortable but slow. It's definitely slower than using the laptop touchpad for zooming on x3d scenes in non-VR mode.

Reply:
I think it makes sense to calibrate to a room size as default, eg. adjust the default speed by a small factor. For a large scene, the navigation speed could be expected to be adjusted by the user or author.
I think the comparison would be between non-VR fly or walk mode, and VR gamepad forward motion ? To my mind there is no real equivalent to examine mode in VR ?
I think examine in VR would involve using your hands to manipulate objects or perhaps the whole world.

Reply:
@andreasplesch  I agree about examine mode in VR. I think we should proceed with the the pull request for now.